ksql-engine/src/main/java/io/confluent/ksql/util/SerDeUtil.java

```diff
@@ -16,81 +16,56 @@
 
 package io.confluent.ksql.util;
 
-import io.confluent.ksql.ddl.DdlConfig;
-import io.confluent.ksql.metastore.MetastoreUtil;
 import io.confluent.ksql.GenericRow;
+import io.confluent.ksql.KsqlEngine;
 import io.confluent.ksql.serde.KsqlTopicSerDe;
-import io.confluent.ksql.serde.avro.KsqlAvroTopicSerDe;
-import io.confluent.ksql.serde.avro.KsqlGenericRowAvroDeserializer;
-import io.confluent.ksql.serde.avro.KsqlGenericRowAvroSerializer;
-import io.confluent.ksql.serde.delimited.KsqlDelimitedDeserializer;
-import io.confluent.ksql.serde.delimited.KsqlDelimitedSerializer;
-import io.confluent.ksql.serde.delimited.KsqlDelimitedTopicSerDe;
-import io.confluent.ksql.serde.json.KsqlJsonDeserializer;
-import io.confluent.ksql.serde.json.KsqlJsonSerializer;
-import io.confluent.ksql.serde.json.KsqlJsonTopicSerDe;
-import org.apache.kafka.common.serialization.Deserializer;
 import org.apache.kafka.common.serialization.Serde;
-import org.apache.kafka.common.serialization.Serdes;
-import org.apache.kafka.common.serialization.Serializer;
 import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
 
-import java.util.HashMap;
-import java.util.Map;
 
 
 public class SerDeUtil {
 
-  public static Serde<GenericRow> getGenericRowJsonSerde(Schema schema) {
-    Map<String, Object> serdeProps = new HashMap<>();
-    serdeProps.put("JsonPOJOClass", GenericRow.class);
-
-    final Serializer<GenericRow> genericRowSerializer = new KsqlJsonSerializer(schema);
-    genericRowSerializer.configure(serdeProps, false);
-
-    final Deserializer<GenericRow> genericRowDeserializer = new KsqlJsonDeserializer(schema);
-    genericRowDeserializer.configure(serdeProps, false);
-
-    return Serdes.serdeFrom(genericRowSerializer, genericRowDeserializer);
-
+  public static Serde<GenericRow> getRowSerDe(final KsqlTopicSerDe topicSerDe, Schema schema,
```

Comment:
Do we need this method anymore? i.e,  now that you can just ask the `KsqlTopicSerde` for the rowSerde, this seems redundant

Reply:
Yes, I removed it.